cpp/order/Steinhardt.h

```diff
@@ -96,28 +100,27 @@ class Steinhardt
         }
     }
 
-    //! Get the last calculated order parameter Wl
-    std::shared_ptr<std::complex<float>> getWl()
-    {
-        return m_WliOrder;
-    }
-
-    //! Get whether the Wl flag was set
-    bool getUseWl()
-    {
-        return m_Wl;
-    }
-
-    //! Get Wl norm
-    std::complex<float> getNormWl()
+    //! Get the last calculated Ql
+    std::shared_ptr<float> getQl()
```

Comment:
This function name is misleading if it can return a Wl value. Is that actually what it should do? I would expect it to return either `m_Qli` or `m_Qlmi`.

Reply:
This was a mistake on my part. Fixed.